sdk/communication/Azure.Communication.CallingServer/src/CallingServerClient.cs

```diff
@@ -220,37 +318,44 @@ public virtual async Task<Response<CallConnection>> JoinCallAsync(string serverC
             }
         }
 
-        /// Join the call using server call id.
-        /// <param name="serverCallId"> The server call id. </param>
-        /// <param name="source"> The source identity. </param>
-        /// <param name="callOptions"> The call Options. </param>
+        /// Create an outgoing call from source to target identities.
+        /// <param name="source"> The source identity </param>
+        /// <param name="targets"> The target identities. </param>
+        /// <param name="callbackUri"> The callback Uri to receive status notifications. </param>
+        /// <param name="options"> The call options. </param>
         /// <param name="cancellationToken"> The cancellation token. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="source"/> is null.</exception>
-        /// <exception cref="ArgumentNullException"><paramref name="serverCallId"/> is null.</exception>
-        /// <exception cref="ArgumentNullException"><paramref name="callOptions"/> is null.</exception>
-        public virtual Response<CallConnection> JoinCall(string serverCallId, CommunicationIdentifier source, JoinCallOptions callOptions, CancellationToken cancellationToken = default)
+        /// <exception cref="ArgumentNullException"><paramref name="targets"/> is null.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="options"/> is null.</exception>
+        public virtual Response<CallConnection> CreateCall(CommunicationIdentifier source, IEnumerable<CommunicationIdentifier> targets, Uri callbackUri, CreateCallOptions options = default, CancellationToken cancellationToken = default)
```

Comment:
Why don't we accept a CallSource object in the method parameters?

Reply:
Good point, created a new object CallSource